### ISSUE TITLE
Delete does not work if not used first

### ISSUE THREAD

**Issue by maxshuty** [alexmacarthur/typeit#248]
If you use the `.delete()` method it does not work if it comes later on in the method chaining (I haven't checked if it is after a specific method call or not). 

[Here](https://codepen.io/maxshuty/pen/dyWKzBj) are two examples, the top one is broken as you can see the text that should be deleted is not deleted, the bottom one works and deletes the text as expected.

Here is the code from the pen above for posterity:

HTML: 
```
<div id="brokenExample"></div>

<br/>

<div id="workingExample"></div>
```
JavaScript:
```
import typeit from "https://cdn.skypack.dev/typeit@7.0.4";

new typeit("#brokenExample")
      .type(`I'm some tet!`)
      .pause(500)
      .move(-2)
      .type('x')
      .pause(250)
      .move(3)
      .break()
      .type('I should be deleted.')
      .pause(500)
      .delete(20)
      .pause(1000)
      .type('I should come after the text above is deleted')
      .go();

new typeit("#workingExample")
      .type('I should be deleted. And I will since I come first...')
      .pause(500)
      .delete(53)
      .pause(1000)
      .type(`I'm some tet!`)
      .pause(500)
      .move(-2)
      .type('x')
      .pause(250)
      .move(3)
      .break()
      .type('Yay! This example works!')
      .go();

**Comment by alexmacarthur** [comment]
Yep.. definitely a bug. Thanks for letting me know!

**Comment by alexmacarthur** [comment]
@maxshuty Fixed! Use v8.0.2 to avoid that bug. Thx again for reporting.
